redis/commands/graph/query_result.py

```diff
@@ -360,3 +360,166 @@ def cached_execution(self):
     @property
     def run_time_ms(self):
         return self._get_stat(INTERNAL_EXECUTION_TIME)
```

Comment:
Should we expose INTERNAL_EXECUTION_TIME to the user with an optional setter? I think this can solve long queries.